### PR TITLE
Project_One

### PR DESCRIPTION
**Pull request by Ironware6**
rafael.perez2@lc.cuny.edu

### CODE REVIEW

**Comment by Ironware6** [comment]
Pull request

File: project_one/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" type="text/css" href="mystyle.css">
+<head>
+	<title></title>
+</head>
+<body>
+    <p id = "pid"> This is section one 
+        <button id = "grading-element-one" onClick = "activate_click()">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        Add A Thing
+        </button> 
+    </p>
+
+    <a href="about.html">About </a>
```
Comment (lehmanclass) [inline project_one/index.html]: redirect works, but cannot find this link with `document.querySelector("grading-element-two")`

File: project_one/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+
+    <div> This is section three
+        <button id = "grading-element-three" onClick = "activate_click_three()">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            Show a thing
+            </button>
+            <div id= "grading-element-three-consequence">
+                Hello I am here.
+            </div>
+    </div>
+
+    <form name = "grading-element-four" method = "POST" action ="http://localhost:3000/form">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+        <input type="text" name="testOne" value="testOne" required/>
+        <input type="text" name="testTwo" value="testTwo" required/>
+        <input type="text" name="testThree" value="testThree" required/>
+
+        <input type="submit" value="Submit">
+ 
+
+    </form>
+    <div id = "grading-element-five">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

**Comment by lehmanclass** [comment]
test #2 almost works, everything else is good. Extra credit options? Mention which one you did, if any, in  a `README.md` file please.

**Comment by Ironware6** [comment]
I pushed to this pull request, my last commit 'fixed final commit' 

Thanks professor. Next time I'll try for extra credit, too many classes right now haha.

**Pull request closed without merging**